src/item_factory.cpp

```diff
@@ -1045,6 +1055,12 @@ void Item_factory::load( islot_comestible &slot, JsonObject &jo )
     }
 }
 
+void Item_factory::load( islot_brewable &slot, JsonObject &jo )
+{
+    slot.time = jo.get_int( "time" );
```

Comment:
Consider expressing `time` as hours in JSON similar to `spoils` with a multiplier here when read